src/HomePage/NavBar/NavBar.scss

```diff
@@ -10,22 +10,28 @@
 }
 
 .navbar {
-  margin: 10px;
+  margin: 5px auto;
   display: flex;
-  justify-content: space-between;
-}
+  flex-direction: row;
+  font-family: $title-font-family;
+  font-size: 16px;
+  width: 500px;
+  justify-content: space-around;
+
+  @media (max-width: 500px) {
+    width: 450px;
+  }
 
-ul {
-  li {
-    padding: 0 20px 0 20px;
-    display: inline;
-    font-family: $title-font-family;
-    font-weight: 300;
-    font-size: 16px;
+  &__link {
+    text-decoration: none;
+    color: black;
```

Comment:
Im intrigued as to why the default colour of your text isn't black when all the text on your site is black 🤔

Reply:
Good point, I'll change it now